openwisp_radius/api/permissions.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from django.utils.translation import gettext_lazy as _
+from rest_framework.exceptions import APIException, PermissionDenied
+from rest_framework.permissions import BasePermission
+
+logger = logging.getLogger(__name__)
+
+
+class HasSmsVerification(BasePermission):
+    def has_permission(self, request, view):
+        """
+        raises PermissionDenied if organization has disabled sms_verification
+        """
+        organization = getattr(view, 'organization')
+        verification = self.has_sms_verification(view, organization)
+        if not verification:
+            message = _(
+                f'View {view.__class__.__name__} is being accessed for organization '
+                f'{organization.name} but SMS verification is disabled for '
+                'this organization.'
+            )
+            logger.warning(message)
+            raise PermissionDenied(message)
```

Comment:
here we can use a translatable string, like `SMS verification is not enabled for this organization`.